web-app/js/ng-app/book/bookController.js

```diff
@@ -1,4 +1,31 @@
-angular.module('ang')
-    .controller('bookController', function ($scope, $http, bookService) {
-        $scope.books = bookService.getAllBooks($http)
-    });
\ No newline at end of file
+angular.module('ang').controller('bookController', [
+    '$route',
+    'bookService',
+    bookController
+]);
+
+function bookController($route, bookService) {
+    var vm = this;
+    vm.reverse = false;
```

Comment:
empty line after variable declaration.

Reply:
According to style guide rename controller to book.controller.js? also check another files.
Avoid camel case in filenames. Only dot notation.